hyperliquid/info.py

```diff
@@ -492,6 +492,56 @@ def user_fees(self, address: str) -> Any:
             }
         """
         return self.post("/info", {"type": "userFees", "user": address})
+    
+    def user_staking_summary(self, address: str) -> Any:
+        """Retrieve the staking summary associated with a user.
+        POST /info
+        Args:
+            address (str): Onchain address in 42-character hexadecimal format;
+                            e.g. 0x0000000000000000000000000000000000000000.
+        Returns:
+            {
+                delegated: float string,
+                undelegated: float string,
+                totalPendingWithdrawal: float string,
+                nPendingWithdrawals: int
+            }
+        """
+        return self.post("/info", {"type": "delegatorSummary", "user": address})
+    
+    def user_stakes(self, address: str) -> Any:
+        """Retrieve the staking info for each validator associated with a user.
```

Comment:
This doesn't return the info for the validator, just the delegation from the user
```suggestion
        """Retrieve the user's staking delegations.
```